pkg/generator/ingress_translator.go

```diff
@@ -58,18 +61,21 @@ type IngressTranslator struct {
 	secretGetter    func(ns, name string) (*corev1.Secret, error)
 	endpointsGetter func(ns, name string) (*corev1.Endpoints, error)
 	serviceGetter   func(ns, name string) (*corev1.Service, error)
+	namespaceGetter func(name string) (*corev1.Namespace, error)
 	tracker         tracker.Interface
 }
```

Comment:
Not for this PR, but it feels weird for this to be a struct rather than an interface.